cmd/template/cluster/flag.go

```diff
@@ -438,6 +439,8 @@ func (f *flag) Validate() error {
 			// skip release validation
 		} else if f.Provider == "gcp" {
 			// skip release validation
+		} else if f.Provider == "capa" {
```

Comment:
[We have a function](https://github.com/giantswarm/kubectl-gs/blob/27079c8851cdb1bcc233bc50657615e2b67b38b7/internal/key/key.go#L112-L115) to detect if it's a CAPI installation. It would be better to add `capa` to that list and re-use the function.